stylus/ui-components/button.styl

```diff
@@ -74,32 +74,44 @@ $button--danger
     color             white
 
     &:active
-    &:hover
+    &:not([disabled]):not([aria-disabled=true]):hover
```

Comment:
@lespacedunmatin As a disabled input could never have focus or be active, I think we don't need to apply them to this selector.